src/CodeGeneration.Roslyn.Tool/Program.cs

```diff
@@ -39,9 +40,9 @@ private static async Task<int> Core(string[] args, CancellationToken cancellatio
             IReadOnlyList<string> refs = Array.Empty<string>();
             IReadOnlyList<string> preprocessorSymbols = Array.Empty<string>();
             IReadOnlyList<string> plugins = Array.Empty<string>();
-            string generatedCompileItemFile = null;
-            string outputDirectory = null;
-            string projectDir = null;
+            string? generatedCompileItemFile = null;
+            string outputDirectory = null!;
+            string projectDir = null!;
```

Comment:
These two assignments are straight-up violations of NRT. No excuse for them, IMO. Please use `string?` here, or explain why it can't be done.

Reply:
Good catch. Based on the `require` parameter in `syntax.DefineOption`, I implicitly derived that the "command line" functionality will check for `null`.
I will add null checks.

Reply:
I added a comment to each assignment. And an explicit null check.
@AArnott Is this OK for you?